### ISSUE TITLE
Discussion: Is this package deprecated/no longer supposed to work anymore?

### ISSUE THREAD

**Issue by tHBp** [Glavin001/atom-beautify#1467]
# Description

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue

**Comment by prettydiff** [comment]
We have been getting a massive number of failure reports lately.  Please see if one of these reported error messages matches what you are seeing: #1462 

If you are seeing something completely different please reopen this issue with an error report so that we can investigate.

**Comment by tHBp** [comment]
> If you're wondering why there are 416 open issues, it is because of issues like this which are uninformative and therefore unproductive, which slow us down and take up a lot of our very limited bandwidth.

Sorry for being non-descriptive about the issue.
I had an issue about beautifying editor contents when I was not able to beautify as an error would throw saying cannot find module. This was a almost 6 weeks ago and I saw failing build status due to which I reverted back to an older revision of the package. 
Sadly, I cannot reproduce the same issue for detailed logs as the latest version 0.29.16 fails on installation for me (same as #1462 ) as @prettydiff commented.
I sincerely apologize for wasting your time.
Thanks.
